packages/workspace/src/browser/workspace-input-dialog.ts

```diff
@@ -49,8 +49,12 @@ export class WorkspaceInputDialog extends SingleTextInputDialog {
         const icon = document.createElement('i');
         icon.classList.add(...codiconArray('folder'));
         icon.style.marginRight = '0.5em';
+        icon.style.verticalAlign = 'middle';
         element.appendChild(icon);
-        element.appendChild(document.createTextNode(label));
+        const path = document.createElement('span');
+        path.style.verticalAlign = 'middle';
+        path.textContent = label;
+        element.appendChild(path);
```

Comment:
This does the job, but I believe you can achieve the same without creating the `span` by adding only.

```typescript
element.style.verticalAlign = 'middle';
icon.style.verticalAlign = 'middle';
```

Is the span otherwise useful?